### ISSUE TITLE
Problem with USB ports

### ISSUE THREAD

**Issue by AlexanderZhirov** [getsolus/packages#1465]
### Summary

It is suspected that there are problems with working with USB ports in the latest kernel build `6.6.12-271.current`. I wrote about it [here](https://discuss.getsol.us/d/10252-problems-with-the-operation-of-the-usb-hub). From under the Live disk, where the kernel version `6.6.9-269.current` USB ports work stably.

```
янв 28 19:01:35 solus kernel: usb 6-4.1: new SuperSpeed USB device number 3 using xhci_hcd
янв 28 19:01:35 solus kernel: usb 6-4.1: New USB device found, idVendor=0781, idProduct=5583, bcdDevice= 1.00
янв 28 19:01:35 solus kernel: usb 6-4.1: New USB device strings: Mfr=1, Product=2, SerialNumber=3
янв 28 19:01:35 solus kernel: usb 6-4.1: Product:  SanDisk 3.2Gen1
янв 28 19:01:35 solus kernel: usb 6-4.1: Manufacturer:  USB
янв 28 19:01:35 solus kernel: usb 6-4.1: SerialNumber: 040198cdcb23853a72d7183dbbb5fc8ebdf34e2b13075526dbb31f3122cc098eb9b50000000000000000000000f3e5d7ff9584188355810786ac7a15
янв 28 19:01:35 solus kernel: usb-storage 6-4.1:1.0: USB Mass Storage device detected
янв 28 19:01:35 solus kernel: scsi host4: usb-storage 6-4.1:1.0
янв 28 19:01:35 solus mtp-probe[2313]: checking bus 6, device 3: "/sys/devices/pci0000:00/0000:00:1c.1/0000:03:00.0/usb6/6-4/6-4.1"
янв 28 19:01:35 solus mtp-probe[2313]: bus: 6, device: 3 was not an MTP device
янв 28 19:01:35 solus mtp-probe[2314]: checking bus 6, device 3: "/sys/devices/pci0000:00/0000:00:1c.1/0000:03:00.0/usb6/6-4/6-4.1"
янв 28 19:01:35 solus mtp-probe[2314]: bus: 6, device: 3 was not an MTP device
янв 28 19:01:36 solus kernel: scsi 4:0:0:0: Direct-Access      USB      SanDisk 3.2Gen1 1.00 PQ: 0 ANSI: 6
янв 28 19:01:36 solus kernel: sd 4:0:0:0: [sdd] 120164352 512-byte logical blocks: (61.5 GB/57.3 GiB)
янв 28 19:01:36 solus kernel: sd 4:0:0:0: [sdd] Write Protect is off
янв 28 19:01:36 solus kernel: sd 4:0:0:0: [sdd] Write cache: disabled, read cache: enabled, doesn't support DPO or FUA
янв 28 19:01:36 solus kernel:  sdd: sdd1
янв 28 19:01:36 solus kernel: sd 4:0:0:0: [sdd] Attached SCSI removable disk
янв 28 19:01:48 solus nemo[2319]: Current gtk theme is not known to have nemo support (Materia-compact) - checking...
янв 28 19:01:52 solus kernel: FAT-fs (sdd1): utf8 is not a recommended IO charset for FAT filesystems, filesystem will be case sensitive!
янв 28 19:01:52 solus udisksd[641]: Mounted /dev/sdd1 at /run/media/alexader/D728-1A01 on behalf of uid 1000
янв 28 19:01:52 solus kernel: DMAR: DRHD: handling fault status reg 2
янв 28 19:01:52 solus kernel: DMAR: [DMA Read NO_PASID] Request device [03:00.0] fault addr 0xfff96000 [fault reason 0x06] PTE Read access is not set
янв 28 19:02:27 solus kernel: hub 6-4:1.0: hub_ext_port_status failed (err = -110)
янв 28 19:02:28 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:29 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:30 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:31 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:32 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:32 solus kernel: usb 6-4-port1: Cannot enable. Maybe the USB cable is bad?
янв 28 19:02:33 solus kernel: usb 6-4-port1: cannot disable (err = -110)
янв 28 19:02:38 solus kernel: hub 6-4:1.0: hub_ext_port_status failed (err = -110)
янв 28 19:02:39 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:41 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:42 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:43 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:44 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:44 solus kernel: usb 6-4-port1: Cannot enable. Maybe the USB cable is bad?
янв 28 19:02:45 solus kernel: usb 6-4-port1: cannot disable (err = -110)
янв 28 19:02:50 solus kernel: hub 6-4:1.0: hub_ext_port_status failed (err = -110)
янв 28 19:02:51 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:52 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:52 solus systemd-udevd[416]: sdd1: Worker [2353] processing SEQNUM=3835 is taking a long time
янв 28 19:02:53 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:54 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:55 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:02:55 solus kernel: usb 6-4-port1: Cannot enable. Maybe the USB cable is bad?
янв 28 19:02:56 solus kernel: usb 6-4-port1: cannot disable (err = -110)
янв 28 19:03:01 solus kernel: hub 6-4:1.0: hub_ext_port_status failed (err = -110)
янв 28 19:03:02 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:03:03 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:03:04 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:03:05 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:03:06 solus kernel: usb 6-4-port1: cannot reset (err = -110)
янв 28 19:03:06 solus kernel: usb 6-4-port1: Cannot enable. Maybe the USB cable is bad?
янв 28 19:03:07 solus kernel: usb 6-4-port1: cannot disable (err = -110)
янв 28 19:03:08 solus kernel: usb 6-4-port1: cannot disable (err = -110)
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 FAILED Result: hostbyte=DID_TIME_OUT driverbyte=DRIVER_OK cmd_age=76s
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 CDB: Read(10) 28 00 00 00 09 08 00 00 f8 00
янв 28 19:03:08 solus kernel: I/O error, dev sdd, sector 2312 op 0x0:(READ) flags 0x80700 phys_seg 248 prio class 2
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 FAILED Result: hostbyte=DID_ERROR driverbyte=DRIVER_OK cmd_age=0s
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 CDB: Read(10) 28 00 00 00 10 00 00 00 08 00
янв 28 19:03:08 solus kernel: I/O error, dev sdd, sector 4096 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 FAILED Result: hostbyte=DID_ERROR driverbyte=DRIVER_OK cmd_age=0s
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 CDB: Read(10) 28 00 00 00 0a 08 00 00 26 00
янв 28 19:03:08 solus kernel: I/O error, dev sdd, sector 2568 op 0x0:(READ) flags 0x80700 phys_seg 38 prio class 2
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 FAILED Result: hostbyte=DID_ERROR driverbyte=DRIVER_OK cmd_age=0s
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 CDB: Read(10) 28 00 00 00 10 00 00 00 08 00
янв 28 19:03:08 solus kernel: I/O error, dev sdd, sector 4096 op 0x0:(READ) flags 0x0 phys_seg 8 prio class 2
янв 28 19:03:08 solus kernel: Buffer I/O error on dev sdd1, logical block 2048, async page read
янв 28 19:03:08 solus kernel: Buffer I/O error on dev sdd1, logical block 2049, async page read
янв 28 19:03:08 solus kernel: Buffer I/O error on dev sdd1, logical block 2050, async page read
янв 28 19:03:08 solus kernel: Buffer I/O error on dev sdd1, logical block 2051, async page read
янв 28 19:03:08 solus kernel: Buffer I/O error on dev sdd1, logical block 2052, async page read
янв 28 19:03:08 solus kernel: Buffer I/O error on dev sdd1, logical block 2053, async page read
янв 28 19:03:08 solus kernel: Buffer I/O error on dev sdd1, logical block 2054, async page read
янв 28 19:03:08 solus kernel: Buffer I/O error on dev sdd1, logical block 2055, async page read
янв 28 19:03:08 solus systemd[844]: Created slice User Background Tasks Slice.
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 FAILED Result: hostbyte=DID_ERROR driverbyte=DRIVER_OK cmd_age=0s
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 CDB: Read(10) 28 00 00 00 09 08 00 00 01 00
янв 28 19:03:08 solus kernel: I/O error, dev sdd, sector 2312 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
янв 28 19:03:08 solus kernel: FAT-fs (sdd1): FAT read failed (blocknr 264)
янв 28 19:03:08 solus kernel: sdd: detected capacity change from 120164352 to 0
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 access beyond end of device
янв 28 19:03:08 solus kernel: I/O error, dev sdd, sector 2312 op 0x0:(READ) flags 0x80700 phys_seg 248 prio class 2
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 access beyond end of device
янв 28 19:03:08 solus kernel: I/O error, dev sdd, sector 2568 op 0x0:(READ) flags 0x80700 phys_seg 38 prio class 2
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 access beyond end of device
янв 28 19:03:08 solus kernel: I/O error, dev sdd, sector 2312 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
янв 28 19:03:08 solus kernel: FAT-fs (sdd1): FAT read failed (blocknr 264)
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 access beyond end of device
янв 28 19:03:08 solus kernel: I/O error, dev sdd, sector 2312 op 0x0:(READ) flags 0x80700 phys_seg 248 prio class 2
янв 28 19:03:08 solus systemd[844]: Starting Cleanup of User's Temporary Files and Directories...
янв 28 19:03:08 solus nemo.desktop[2319]: Couldn't get disk full info for: Произошла ошибка при получении сведений о файловой системе /run/media/alexader/D728-1A01: Ошибка ввода/вывода
янв 28 19:03:08 solus systemd[844]: Finished Cleanup of User's Temporary Files and Directories.
янв 28 19:03:08 solus nemo.desktop[2319]: Couldn't get disk full info for: Произошла ошибка при получении сведений о файловой системе /run/media/alexader/D728-1A01: Ошибка ввода/вывода
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 access beyond end of device
янв 28 19:03:08 solus kernel: I/O error, dev sdd, sector 2568 op 0x0:(READ) flags 0x80700 phys_seg 38 prio class 2
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 access beyond end of device
янв 28 19:03:08 solus kernel: FAT-fs (sdd1): FAT read failed (blocknr 264)
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 access beyond end of device
янв 28 19:03:08 solus kernel: FAT-fs (sdd1): FAT read failed (blocknr 1708)
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 access beyond end of device
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 access beyond end of device
янв 28 19:03:08 solus nemo.desktop[2319]: Couldn't get disk full info for: Произошла ошибка при получении сведений о файловой системе /run/media/alexader/D728-1A01: Ошибка ввода/вывода
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 access beyond end of device
янв 28 19:03:08 solus kernel: FAT-fs (sdd1): FAT read failed (blocknr 264)
янв 28 19:03:08 solus udisksd[641]: Partitions found on device '/dev/sdd' but couldn't read partition table signature: Failed to assign the new context to disk '/dev/sdd': No medium found
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 access beyond end of device
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 access beyond end of device
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 access beyond end of device
янв 28 19:03:08 solus kernel: FAT-fs (sdd1): FAT read failed (blocknr 264)
янв 28 19:03:08 solus udisksd[641]: Cleaning up mount point /run/media/alexader/D728-1A01 (device 8:49 no longer exists)
янв 28 19:03:08 solus kernel: sd 4:0:0:0: [sdd] tag#0 access beyond end of device
янв 28 19:03:08 solus kernel: Buffer I/O error on dev sdd1, logical block 0, lost sync page write
янв 28 19:03:08 solus systemd[1]: run-media-alexader-D728\x2d1A01.mount: Deactivated successfully.
янв 28 19:03:13 solus kernel: hub 6-4:1.0: hub_ext_port_status failed (err = -110)
```

### Steps to reproduce

No data

### Expected result

No data

### Actual result

No data

### Environment

- [x] Is system up to date?

### Repo

Shannon (stable)

### Desktop Environment

Budgie

### System details

```
System:
  Host: solus Kernel: 6.6.12-271.current arch: x86_64 bits: 64 Desktop: Budgie
    v: 10.8.2 Distro: Solus 4.5 resilience
Machine:
  Type: Desktop System: Gigabyte product: N/A v: N/A
    serial: <superuser required>
  Mobo: Gigabyte model: Z77-DS3H v: x.x serial: <superuser required>
    UEFI: American Megatrends v: F9 date: 09/19/2012
CPU:
  Info: quad core Intel Core i7-3770 [MT MCP] speed (MHz): avg: 1694
    min/max: 1600/3900
Graphics:
  Device-1: NVIDIA GP107 [GeForce GTX 1050 Ti] driver: nvidia v: 535.154.05
  Device-2: Logitech Webcam C310 driver: snd-usb-audio,uvcvideo type: USB
  Display: x11 server: X.Org v: 21.1.11 with: Xwayland v: 23.2.4 driver:
    gpu: nvidia,nvidia-nvswitch resolution: 1: 2560x1080~60Hz 2: 1440x900~60Hz
  API: OpenGL v: 4.6.0 vendor: nvidia v: 535.154.05 renderer: NVIDIA
    GeForce GTX 1050 Ti/PCIe/SSE2
Network:
  Device-1: Qualcomm Atheros AR8151 v2.0 Gigabit Ethernet driver: atl1c
Drives:
  Local Storage: total: 5.76 TiB used: 322.12 GiB (5.5%)
Info:
  Processes: 354 Uptime: 2h 52m Memory: total: 32 GiB available: 31.32 GiB
  used: 5.55 GiB (17.7%) Shell: Bash inxi: 3.3.31
  ```

**Comment by ermo** [comment]
I can confirm seeing this. I thought it was just my Kobo e-reader going bad, but maybe it isn't?

**Comment by AlexanderZhirov** [comment]
Updating the kernel to version 6.6.15-272.current did not solve the problem with USB operation

```
фев 03 13:22:36 solus kernel: usb 6-4.3: new SuperSpeed USB device number 3 using xhci_hcd
фев 03 13:22:36 solus kernel: usb 6-4.3: New USB device found, idVendor=0781, idProduct=5583, bcdDevice= 1.00
фев 03 13:22:36 solus kernel: usb 6-4.3: New USB device strings: Mfr=1, Product=2, SerialNumber=3
фев 03 13:22:36 solus kernel: usb 6-4.3: Product:  SanDisk 3.2Gen1
фев 03 13:22:36 solus kernel: usb 6-4.3: Manufacturer:  USB
фев 03 13:22:36 solus kernel: usb 6-4.3: SerialNumber: 040198cdcb23853a72d7183dbbb5fc8ebdf34e2b13075526dbb31f3122cc098eb9b50000000000000000000000f3e5d7ff9584188355810786ac7a15
фев 03 13:22:36 solus mtp-probe[1737]: checking bus 6, device 3: "/sys/devices/pci0000:00/0000:00:1c.1/0000:03:00.0/usb6/6-4/6-4.3"
фев 03 13:22:36 solus mtp-probe[1737]: bus: 6, device: 3 was not an MTP device
фев 03 13:22:36 solus kernel: usb-storage 6-4.3:1.0: USB Mass Storage device detected
фев 03 13:22:36 solus kernel: scsi host4: usb-storage 6-4.3:1.0
фев 03 13:22:36 solus kernel: usbcore: registered new interface driver usb-storage
фев 03 13:22:36 solus kernel: usbcore: registered new interface driver uas
фев 03 13:22:36 solus mtp-probe[1757]: checking bus 6, device 3: "/sys/devices/pci0000:00/0000:00:1c.1/0000:03:00.0/usb6/6-4/6-4.3"
фев 03 13:22:36 solus mtp-probe[1757]: bus: 6, device: 3 was not an MTP device
фев 03 13:22:37 solus kernel: scsi 4:0:0:0: Direct-Access      USB      SanDisk 3.2Gen1 1.00 PQ: 0 ANSI: 6
фев 03 13:22:37 solus kernel: sd 4:0:0:0: [sdd] 120164352 512-byte logical blocks: (61.5 GB/57.3 GiB)
фев 03 13:22:37 solus kernel: sd 4:0:0:0: [sdd] Write Protect is off
фев 03 13:22:37 solus kernel: sd 4:0:0:0: [sdd] Write cache: disabled, read cache: enabled, doesn't support DPO or FUA
фев 03 13:22:37 solus kernel:  sdd: sdd1
фев 03 13:22:37 solus kernel: sd 4:0:0:0: [sdd] Attached SCSI removable disk
фев 03 13:22:45 solus kernel: FAT-fs (sdd1): utf8 is not a recommended IO charset for FAT filesystems, filesystem will be case sensitive!
фев 03 13:22:45 solus kernel: FAT-fs (sdd1): Volume was not properly unmounted. Some data may be corrupt. Please run fsck.
фев 03 13:22:45 solus udisksd[733]: Mounted /dev/sdd1 at /run/media/alexander/D728-1A01 on behalf of uid 1000
фев 03 13:22:45 solus kernel: DMAR: DRHD: handling fault status reg 2
фев 03 13:22:45 solus kernel: DMAR: [DMA Read NO_PASID] Request device [03:00.0] fault addr 0xfffc2000 [fault reason 0x06] PTE Read access is not set
фев 03 13:22:45 solus kernel: xhci_hcd 0000:03:00.0: WARNING: Host System Error
фев 03 13:22:46 solus kernel: xhci_hcd 0000:03:00.0: Host halt failed, -110
фев 03 13:23:21 solus kernel: xhci_hcd 0000:03:00.0: xHCI host not responding to stop endpoint command
фев 03 13:23:21 solus kernel: xhci_hcd 0000:03:00.0: Host halt failed, -110
фев 03 13:23:21 solus kernel: xhci_hcd 0000:03:00.0: xHCI host controller not responding, assume dead
фев 03 13:23:21 solus kernel: xhci_hcd 0000:03:00.0: HC died; cleaning up
фев 03 13:23:21 solus kernel: usb 5-1: USB disconnect, device number 2
фев 03 13:23:21 solus kernel: usb 5-1.4: USB disconnect, device number 3
фев 03 13:23:21 solus kernel: sd 4:0:0:0: [sdd] tag#0 FAILED Result: hostbyte=DID_TIME_OUT driverbyte=DRIVER_OK cmd_age=35s
фев 03 13:23:21 solus kernel: sd 4:0:0:0: [sdd] tag#0 CDB: Read(10) 28 00 07 29 8e d8 00 00 08 00
фев 03 13:23:21 solus kernel: I/O error, dev sdd, sector 120164056 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
фев 03 13:23:21 solus kernel: sd 4:0:0:0: [sdd] tag#0 FAILED Result: hostbyte=DID_ERROR driverbyte=DRIVER_OK cmd_age=0s
фев 03 13:23:21 solus kernel: sd 4:0:0:0: [sdd] tag#0 CDB: Read(10) 28 00 00 00 08 2f 00 00 11 00
фев 03 13:23:21 solus kernel: I/O error, dev sdd, sector 2095 op 0x0:(READ) flags 0x80700 phys_seg 17 prio class 2
фев 03 13:23:21 solus kernel: usb 6-4: USB disconnect, device number 2
фев 03 13:23:21 solus kernel: usb 6-4.3: USB disconnect, device number 3
фев 03 13:23:21 solus nemo.desktop[1694]: Couldn't get disk full info for: Error getting filesystem info for /run/media/alexander/D728-1A01: Input/output error
фев 03 13:23:21 solus kernel: device offline error, dev sdd, sector 2120 op 0x0:(READ) flags 0x80700 phys_seg 56 prio class 2
фев 03 13:23:21 solus kernel: device offline error, dev sdd, sector 120164056 op 0x0:(READ) flags 0x0 phys_seg 8 prio class 2
фев 03 13:23:21 solus kernel: Buffer I/O error on dev sdd1, logical block 120162008, async page read
фев 03 13:23:21 solus kernel: Buffer I/O error on dev sdd1, logical block 120162009, async page read
фев 03 13:23:21 solus kernel: Buffer I/O error on dev sdd1, logical block 120162010, async page read
фев 03 13:23:21 solus kernel: Buffer I/O error on dev sdd1, logical block 120162011, async page read
фев 03 13:23:21 solus kernel: Buffer I/O error on dev sdd1, logical block 120162012, async page read
фев 03 13:23:21 solus kernel: Buffer I/O error on dev sdd1, logical block 120162013, async page read
фев 03 13:23:21 solus kernel: Buffer I/O error on dev sdd1, logical block 120162014, async page read
фев 03 13:23:21 solus kernel: Buffer I/O error on dev sdd1, logical block 120162015, async page read
фев 03 13:23:21 solus kernel: FAT-fs (sdd1): FAT read failed (blocknr 46)
фев 03 13:23:21 solus udisksd[733]: Cleaning up mount point /run/media/alexander/D728-1A01 (device 8:49 no longer exists)
фев 03 13:23:21 solus systemd[1]: run-media-alexander-D728\x2d1A01.mount: Deactivated successfully.
фев 03 13:23:21 solus udisksd[733]: Couldn't find existing drive object for device /sys/devices/pci0000:00/0000:00:1c.1/0000:03:00.0/usb6/6-4/6-4.3/6-4.3:1.0/host4/target4:0:0/4:0:0:0/block/sdd (uevent action 'change', VPD 'pci-0000:03:00.0-usb-0:4.3')
```

**Comment by ermo** [comment]
I cannot reproduce this issue on kernel 6.8.x in Solus.

**Comment by AlexanderZhirov** [comment]
@ermo I will try to test it in the near future.

**Comment by ermo** [comment]
@AlexanderZhirov ping? Any news on this?

**Comment by ermo** [comment]
Unless the reporter tells us that this is still an issue on 6.6.x LTS, this issue is now low priority.

**Comment by ermo** [comment]
Fixed in 6.6.31-245.lts for me, closing.

@AlexanderZhirov Please re-open if this is still a problem for you.
